Toolchain/nosman/src/main.rs

```diff
@@ -114,6 +114,11 @@ fn main() {
                 .num_args(0)
                 .required(false)
             )
+            .arg(Arg::new("no-dependencies")
+                .long("no-dependencies")
```

Comment:
I guess --without-deps could be a better name:
`nodos install acme.some.module --without-deps`